core/src/main/java/com/netflix/iceberg/BaseMetastoreTableOperations.java

```diff
@@ -167,18 +159,6 @@ private static String newMetadataLocation(String baseLocation, String filename)
     return String.format("%s/%s/%s", baseLocation, METADATA_FOLDER_NAME, filename);
   }
 
-  private static String parseBaseLocation(String metadataLocation) {
```

Comment:
Note to myself: this removal is good because it appears to be an orphaned method.